src/beethovenx/components/images/AppIcon.vue

```diff
@@ -1,19 +1,30 @@
 <template>
   <img
-    src="~@/beethovenx/assets/images/logo.svg"
-    width="175"
-    class="pl-2 xl:pl-6 "
+    v-if="!showSmallLogo"
+    src="~@/beethovenx/assets/images/logo-full.svg"
+    width="180"
+  />
+  <img
+    v-if="showSmallLogo"
+    src="~@/beethovenx/assets/images/logo-mbl.svg"
+    width="99"
   />
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent, computed } from 'vue';
+import useBreakpoints from '@/composables/useBreakpoints';
 
 export default defineComponent({
   name: 'AppIcon',
 
   setup() {
-    return {};
+    const { bp } = useBreakpoints();
+    const showSmallLogo = computed(() => {
+      return bp.value === 'xs' || bp.value === 'lg';
```

Comment:
Something like this might also work: 
`computed(() => ['xs', 'lg'].includes(bp.value));`